HabitRPG/Utilities/AuthenticationManager.swift

```diff
@@ -76,6 +80,7 @@ class AuthenticationManager: NSObject {
     func setAuthentication(userId: String, key: String) {
         currentUserId = userId
         currentUserKey = key
+        localKeychain[userId] = key
```

Comment:
Is this actually needed? setting the currentUserKey seems to also be setting the localKeychain

Reply:
Sorry, I left it in there when I was doing testing, so I could just launch the main app to store the credentials and not log in-out.

Reply:
It's been removed in a recent commit